### DISCUSSION TITLE
Service not started with Windows (installed)

### DISCUSSION THREAD

**Discussion by cubicYYY** [rustdesk/rustdesk#6620] [General]
### Bug Description

When the app is launched, the service is not.
I am trying to make the RustDesk start with Windows (along with the service)
However, I found that only after manually clicking the button start the service can I see it in services.msc:
![image](https://github.com/rustdesk/rustdesk/assets/93425234/052056e6-36d1-456c-9a2a-29ff86957aff)
I try to start it with `"C:\Program Files\RustDesk\rustdesk.exe" --service` but there's no service running.

### How to Reproduce

Installed and run

### Expected Behavior

Start with Windows

### Operating system(s) on local side and remote side

Win11 23H2

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/93425234/788f5b86-d53f-4c8d-80ff-46d8759b433f)

### Additional Context

Is there any way to start the service in command line? So that I can set it into scheduled tasks to start with Windows.

**Comment by cubicYYY** [comment]
Install->Start the service->Restart: OK
Install->Start the service->Close the application->Restart: Will not start the service

**Comment by gravelyt** [comment]
Got exactly the same problem.

**Comment by TheNerd** [comment]
I had this issue as well and found a solution on reddit [here](https://www.reddit.com/r/rustdesk/comments/172grjw/comment/kuofhd5/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button). 

> As soon as you tell it to run the Service, it should boot with the Service next time UNLESS you Exit by right-clicking the taskbar icon and selecting Exit. If you do that, the Service also REMOVES itself as opposed to just disabling.

So basically, leave the application running and when you reboot it should start fine. I just tested this on Windows 11 and it worked for me. The post contains tons more info if you still have issues.
